### PR TITLE
Update parent dashboard to new UI

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-13014
affects: Parent
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020089&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020108&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020109&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-13014](https://instructure.atlassian.net/browse/MBL-13014)
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020110&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020114&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020119&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020232&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020233&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020407&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: Parent/Parent/Dashboard/DashboardViewController.swift [hunk 1]
```diff
@@ -204,7 +182,7 @@ class DashboardViewController: UIViewController {
         try retrieveStudents()
     }
 
-    @objc func retrieveStudents() throws {
+    func retrieveStudents() throws {
```
[the remark this answers is not in the document]
Comment (bluwave) [reply Parent/Parent/Dashboard/DashboardViewController.swift]: Why don't i make a new ticket for that.  I'd rather not do it in this pr if possible.

[the remark this answers is not in the document]
Comment (bluwave) [reply Parent/Parent/Dashboard/DashboardViewController.swift]: MBL-13921

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020408&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020409&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

File: Parent/Parent/Dashboard/DashboardViewController.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (shichongrui) [reply Parent/Parent/Dashboard/DashboardViewController.swift]: 👍

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by bluwave** [comment]
icon fixed!

> Also pull to refresh does not update the inbox # badge on the hamburger icon.
I really didn't change anything about the fetching of inbox count.  This is a little bit tricky here, b/c DashboardViewController is just the container for all the tabs and stuff.  The pull to refresh is on the GradesViewController.

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020413&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1020414&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|1
Medium|23
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---
High|1

**Pull request merged**